<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CEthernetCfgForm</class>
 <widget class="QWidget" name="CEthernetCfgForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Ethernet</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>350</width>
     <height>3</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_77">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>45</y>
     <width>60</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adres IP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_79">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>105</y>
     <width>120</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Brama domy≈õlna:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipAddrEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>120</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string>000.000.000.000</string>
   </property>
   <property name="text">
    <string>10.1.1.180</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gatewayEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>120</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string>000.000.000.000</string>
   </property>
   <property name="text">
    <string>10.1.1.1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maskEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>120</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string>000.000.000.000</string>
   </property>
   <property name="text">
    <string>255.255.255.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_78">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>110</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Maska podsieci:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
